pkg/template/cloudconfig.go

```diff
@@ -135,15 +135,26 @@ func (c *CloudConfig) String() string {
 	return c.config
 }
 
-func nodeName(index int) string {
-	return fmt.Sprintf("etcd%d", index)
+func etcdClusterSize(highAvailability bool) int {
+	if highAvailability {
+		return 3
```

Comment:
I think it's a bad idea in the long run to hardcode this here. Fine for now.

Reply:
I agree. Might want 5 or 7 in the future. I guess we could pass this in as a parameter rather than the boolean `HighAvailability`. We'll work it out as we go.